GoogleDataTransport/GDTCORLibrary/GDTCORUploadCoordinator.m

```diff
@@ -141,7 +141,8 @@ - (void)uploadTargets:(NSArray<NSNumber *> *)targets conditions:(GDTCORUploadCon
  */
 - (GDTCORUploadConditions)uploadConditions {
 #if TARGET_OS_WATCH
-  return GDTCORUploadConditionNoNetwork;
+  // Assume connected on watchOS.
+  return GDTCORUploadConditionUnclearConnection;
```

Comment:
I don't see this being defined anywhere

Reply:
Previously defined in GDTCORPrioritizer.h as GDTCORUploadConditions.